src/data/api.ts

```diff
@@ -322,6 +322,20 @@ export const GetUserMetadata = async (walletAddr: string) => {
   return tzktData
 }
 
+/**
+ * Get User DAO token balance
+ */
+export async function getDaoTokenBalance(walletAddr: string) {
+  const parameters = {
+    'token.contract': DAO_TOKEN_CONTRACT,
+    'token.tokenId': '0',
+    'account': walletAddr,
+    "select": 'balance'}
+  const response = await axios.get(`https://api.tzkt.io/v1/tokens/balances`, { params: parameters })
+    .catch(error => console.log('Error while querying the account token balance:', error))
```

Comment:
Is this not doing what you want @melMass ?

Reply:
It will return 0 tokens, unless i did something wrong.

Reply:
Can the tzkt url be changed to `VITE_TZKT_API` ? If we ever need to switch tzkt instances, then we only need to change it in https://github.com/teia-community/teia-ui/blob/main/.env

Reply:
Yes indeed, I did not get that the last `?` here is a conditional and not a null property fallback https://github.com/teia-community/teia-ui/blob/364e3c22c79642cecde127e9127f609f726b2045/src/data/api.ts#L337

Are you sure the hooks are running 🤣 ?

Reply:
> Can the tzkt url be changed to `VITE_TZKT_API` ? If we ever need to switch tzkt instances, then we only need to change it in https://github.com/teia-community/teia-ui/blob/main/.env

Did it in my last commit

Reply:
Awesome!